pebbles/views/blueprint_templates.py

```diff
@@ -127,6 +129,22 @@ def put(self, template_id):
         db.session.commit()
 
 
+class BlueprintTemplateCopy(restful.Resource):
+    @auth.login_required
+    @requires_admin
+    def put(self, template_id):
+        template = BlueprintTemplate.query.filter_by(id=template_id).first()
+        if not template:
+            abort(404)
+
+        db.session.expunge(template)
+        make_transient(template)
+        template.id = uuid.uuid4().hex
+        template.name = format("%s - %s" % (template.name, 'Copy'))
```

Comment:
The name doesn't matter, as they are identified by uuid in the id field. If somebody is copying something, he should know what he wants and should change it IMO